ntbs-service-unit-tests/Helpers/EpisodesHelperTest.cs

```diff
@@ -86,11 +95,15 @@ public void GroupEpisodesIntoPeriods_GroupsBasedOnEndingOutcomeType()
             var expectedPeriod4 = TreatmentPeriod.CreateTreatmentPeriod(4, _testTreatmentEvents[6]);
             expectedPeriod4.TreatmentEvents.AddRange(new[] { _testTreatmentEvents[8], _testTreatmentEvents[7] });
 
+            var expectedPeriod5 = TreatmentPeriod.CreateTreatmentPeriod(5, _testTreatmentEvents[10]);
+            expectedPeriod5.TreatmentEvents.AddRange(new[] { _testTreatmentEvents[9] });
+
             AssertTreatmentPeriodMatchesExpected(expectedPeriod0, periods[0]);
             AssertTreatmentPeriodMatchesExpected(expectedPeriod1, periods[1]);
             AssertTreatmentPeriodMatchesExpected(expectedPeriod2, periods[2]);
             AssertTreatmentPeriodMatchesExpected(expectedPeriod3, periods[3]);
             AssertTreatmentPeriodMatchesExpected(expectedPeriod4, periods[4]);
+            AssertTreatmentPeriodMatchesExpected(expectedPeriod5, periods[5]);
```

Comment:
I do think this test has become too large, and it's no longer clear exactly which edge cases it is meant to be testing. It would be nice to break it up into different tests... I'll leave it to you to decide if it's worth the effort or not.

Reply:
Yeah I've split this in two, thanks!